README.md

```diff
@@ -124,6 +124,7 @@ An example configuration file called `sample_configuration.yaml` is provided. Wh
 | publish_topic_names | Whether or not to include topic name information in the log messsages that are uploaded to AWS CloudWatch Logs | *bool* | true/false | true |
 | storage_directory | The location where all offline metrics will be stored | *string* | string | ~/.ros/cwlogs/ |
 | storage_limit | The maximum size of all offline storage files in KB. Once this limit is reached offline logs will start to be deleted oldest first. | *int* | number | 1048576 |
+| discard_stale_data | Whether or not to discard log batch data that are over 14 days old, which are rejected by AWS PutLogEvents. | *bool* | true/false | false |
```

Comment:
```suggestion
| delete_stale_data | Whether or not to delete log batch data that are over 14 days old, which are rejected by AWS PutLogEvents. | *bool* | true/false | false |
```

Let's make it clear that log data over 14 days old will never be uploaded and this flag deletes the data from disk (if it was read from disk).